testing_and_setup/compass/ocean/hurricane/USDEQU120at30cr10rr2/synthetic_sandy/config_init.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<config case="init">
+	<add_link source="../../build_mesh/culled_mesh/culled_mesh.nc" dest="mesh.nc"/>
+	<add_link source="../../build_mesh/culled_mesh/culled_graph.info" dest="graph.info"/>
+	<add_link source_path="script_configuration_dir" source="create_pointstats_file.py" dest="create_pointstats_file.py"/>
+	<add_link source_path="script_configuration_dir" source="create_pointstats_file.py" dest="create_pointstats_file.py"/>
+	<add_link source_path="script_test_dir" source="NOAA-COOPS_stations.txt" dest="NOAA-COOPS_stations.txt"/>
+	<add_link source_path="script_test_dir" source="USGS_stations.txt" dest="USGS_stations.txt"/>
+	<add_link source_path="script_configuration_dir" source="hurricane_wind_pressure/main.py" dest="create_time_varying_forcing.py"/>
+	<add_link source_path="script_configuration_dir" source="spinup_time_varying_forcing.py" dest="spinup_time_varying_forcing.py"/>
+	<add_link source_path="script_test_dir" source="SANDY_017771.json" dest="SANDY_017771.json"/>
+	<add_link source_path="script_test_dir" source="hurricane_inputs.txt" dest="hurricane_inputs.txt"/>
```

Comment:
Typo-- should be <add_link source_path="script_test_dir" source="hurricane_wind_pressure/hurricane_inputs.txt" dest="hurricane_inputs.txt"/>

Reply:
I fixed this by adding this file to the `script_test_dir`, since the inputs (i.e. the json filename) are specific to the testcase